ai-platform-unified/notebooks/unofficial/matching_engine/matching_engine_for_indexing.ipynb

```diff
@@ -32,7 +32,7 @@
         "<table align=\"left\">\n",
         "\n",
         "  <td>\n",
-        "    <a href=\"https://console.cloud.google.com/ai/platform/notebooks/deploy-notebook?download_url=https://github.com/GoogleCloudPlatform/ai-platform-samples/blob/master/ai-platform-unified/notebooks/unofficial/matching_engine/matching_engine_for_indexing.ipynb\"\">\n",
```

Comment:
Can you not use the `https://github.com/GoogleCloudPlatform/ai-platform-samples/blob` version of the link instead of `https://raw.githubusercontent.com/`?

Reply:
Done